src/entrypoints/custom-panel.ts

```diff
@@ -140,3 +140,10 @@ document.addEventListener(
   () => window.parent.customPanel!.registerIframe(initialize, setProperties),
   { once: true }
 );
+
+window.addEventListener("unload", () => {
+  // allow disconnected callback to fire
+  while (document.body.lastChild) {
+    document.body.removeChild(document.body.lastChild);
```

Comment:
This will also remove any `script` and/or `style` tags... is that desired?  And if thy are moved to `head`, it won't.  My gut says it should be more explicit.

Reply:
It will be unloaded anyway, so doesn't really matter 🤷‍♂️